Source/Blazorise.FluentUI2/Styles/components/_card.scss

```diff
@@ -5,8 +5,6 @@
     position: relative;
     display: flex;
     flex-direction: column;
-    overflow-x: hidden;
-    overflow-y: hidden;
```

Comment:
@stsrki Please evaluate this change. 
The problem is if the overflow is not permitted, inner content will not be available to overflow on top of the card.

Reply:
(Autocomple dropdown fix)

Reply:
What is the problem here? Can you clarify?

Reply:
Did I not just say? 
![image](https://github.com/Megabit/Blazorise/assets/22283161/bc9bbda3-17f7-4b8d-8784-c7435f171a86)

Reply:
Yes, but I tried it on my side, and it works. That's why I'm asking for more information.

Reply:
It works due to this change

Reply:
It seems it is not working after all.

![image](https://github.com/Megabit/Blazorise/assets/900302/9cfe0883-a2c8-4bf0-a14d-1202004bac2a)

I will look into it.

Reply:
Ok, so I fixed Autocomplete, and I broke that one then?
Please test Autocomplete with whatever changes you make.

Reply:
It should be fixed now. Autocomplete also works.